### PR TITLE
Fixed REST docs (items for array)

### PR DESCRIPTION
**Pull request by alexeyvy**
According to https://oat-sa.atlassian.net/browse/TAO-6641

### CODE REVIEW

File: doc/rest.json [hunk 1]
```diff
@@ -377,6 +377,9 @@
                                         },
```
Comment (chilly86) [inline doc/rest.json]: in the data I see: id, status, report

Comment (alexeyvy) [reply doc/rest.json]: $result['id']     = $this->getTaskId($taskLogEntity);
        $result['status'] = $this->getTaskStatus($taskLogEntity);
        $result['report'] = $taskLogEntity->getReport() ? $this->getTaskReport($taskLogEntity) : [];

        return array_merge($result, (array) $this->addExtraReturnData($taskLogEntity));

Comment (alexeyvy) [reply doc/rest.json]: addExtraReturnData adds the field `itemIds`

Comment (alexeyvy) [reply doc/rest.json]: And that's strange that swagger editor didn't highlight the error for you. For me it's: ```Semantic error at paths./taoQtiItem/RestQtiItem/getStatus.get.responses.200.schema.properties.data.properties.itemIds
Schemas with 'type: array', require a sibling 'items: ' field
Jump to line 377
```

**Comment by alexeyvy** [comment]
Docs parameters have been fixed!

File: controller/RestQtiItem.php [hunk 2]
```diff
@@ -165,7 +165,7 @@ public function importDeferred()
             $taskLog = $this->getServiceManager()->get(TaskLogInterface::SERVICE_ID);
 
             if ($report = $taskLog->getReport($task->getId())) {
-                $result['report'] = $report->toArray();
+                $result['report'] = $this->getReportAsAssociativeArray($report);
```
Comment (chilly86) [inline controller/RestQtiItem.php]: the two format is different, let's not touch it and keep backward compatibility

Comment (alexeyvy) [reply controller/RestQtiItem.php]: Done :)

**Pull request merged**